app/src/docs/v1.api-spec.yaml

```diff
@@ -0,0 +1,2744 @@
+        '404':
+          $ref: '#/components/responses/NotFound'
+        default:
+          $ref: '#/components/responses/Error'
+  /submission/emailConfirmation:
+    put:
+      summary: Send confirmation Email
+      operationId: emailConfirmation
+      tags:
+        - Submission
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/EmailData'
+      responses:
+        '201':
+          description: Email Confirmation was set
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response-Email'
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        "422":
+          $ref: "#/components/responses/UnprocessableEntity"
+        default:
+          $ref: '#/components/responses/Error'
+  /submission/search:
+    get:
+      summary: Search submissions
+      description: Returns a list of submissions based on the given queries
+      operationId: searchSubmissions
+      tags:
+        - Submission
+      parameters:
+        - $ref: '#/components/parameters/Query-IncludeUser'
+        - in: query
+          name: activityId
+          required: false
+          schema:
+            type: array
+            description: the activity ids to filter on
+            items:
+              $ref: '#/components/parameters/Path-ActivityId'
+        - in: query
+          name: submissionId
+          required: false
+          schema:
+            type: array
+            description: The submissions to filter on
+            items:
+              $ref: '#/components/parameters/Path-SubmissionId'
+        - in: query
+          name: intakeStatus
+          required: false
+          schema:
+            type: array
+            description: The intake statuses to filter on
+            items:
+              $ref: '#/components/parameters/IntakeStatus'
+        - in: query
+          name: submissionType
+          required: false
+          schema:
+            type: array
+            description: The submission types to filter on
+      responses:
+        '200':
+          description: A list of submissions matching the search criteria
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/DB-Submission'
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        "422":
+          $ref: "#/components/responses/UnprocessableEntity"
+        default:
+          $ref: '#/components/responses/Error'
+  /submission/statistics:
+    get:
+      summary: Get submission statistics
+      description: Gets a set of submission related statistics
+      operationId: getStatistics
```

Comment:
This obtains statistics for both submissions and enquiries, despite being under the /submission route